.eslintrc.json

```diff
@@ -19,5 +19,16 @@
         }
       }
     }
-  }
+  },
+  "rules": {
+    "max-lines-per-function": ["error"]
+  },
+  "overrides": [
+    {
+      "files": ["src/components/**", "src/containers/**"],
+      "rules": {
+        "max-lines-per-function": ["error", 120]
```

Comment:
I think we should use the recommended 50 lines max per function.  But omit any functions having to do with rendering.  Thoughts? @shaunmaharaj  @rostyk-kanafotskyy

Use this exemption here.
// eslint-disable-next-line max-lines-per-function

Reply:
@aChanEP That's a good point, but I believe we have functions with more than 50 lines too and it could be a pain to refactor them all due to the multiple conditions we support.

I'm good with 120 as max lines for now, and we can clean up more once the time comes to integrate Cortex JS

Reply:
works for me! could be wasted work once the CortexJs comes in.